Gene.h

```diff
@@ -260,19 +260,19 @@ class Gene{
       int n = (*codonNum) % 3;
       int cdsIdx = i;
       switch(n){
-        case 0:
```

Comment:
I think the original code is correct:

As (*codonNum) is 1-based integer, when it equals to 3, variable n is 0.
Then the chromosomal position for the third letter for the codon is:
codonPos[2] = variantPos

So I think the *case 0* should not be replaced with "case 2" here. 

Does this make sense to you? (Sorry for the lack of documentation in this function).